src/featherlight.gallery.js

```diff
@@ -55,7 +57,9 @@
 					if (swipeAwareConstructor) {
 						self._swiper = swipeAwareConstructor(self.$instance)
 							.on('swipeleft', self._swipeleft = function()  { self.$instance.trigger('next'); })
-							.on('swiperight', self._swiperight = function() { self.$instance.trigger('previous'); });
+							.on('swiperight', self._swiperight = function() { self.$instance.trigger('previous'); })
+							.on('swipeup', self._swipeup = function() { $.featherlight.close(event); })
```

Comment:
should be `function(event) { ... }`, right?

Reply:
right. this is my mistake.